turbo.json

```diff
@@ -1,17 +1,18 @@
 {
-  "$schema": "https://turborepo.org/schema.json",
-  "pipeline": {
-    "typedocs": {
-      "cache": false
-    },
-    "storydocs": {
-      "cache": false
-    },
-    "build": {
-      "outputs": ["dist/**", "styled-system/**"]
-    },
-    "lint": {},
-    "test:ci": {},
-    "typecheck": {}
-  }
+	"$schema": "https://turborepo.org/schema.json",
+	"pipeline": {
+		"typedocs": {
+			"cache": false
+		},
+		"storydocs": {
+			"cache": false
+		},
+		"build": {
+			"outputs": ["dist/**", "styled-system/**"]
+		},
+		"lint": {},
+		"test:ci": {},
+		"typecheck": {},
+		"format": {}
+	}
```

Comment:
I think this shouldn't change indentation

Reply:
Are you talking about in `biome.json`? Should I leave it with the default `tab` style?

Reply:
Yes and no, I mean we use 2 spaces: `"indentWidth": 2,` in `biome.json`

Reply:
Oh, that is already a [default setting](https://biomejs.dev/reference/configuration/#javascriptformatterindentwidth)

Reply:
Okay, I found it's an another option for JSON https://biomejs.dev/reference/configuration/#jsonformatterindentstyle

Reply:
Oh! That's what you meant. 🤦 That's what I get for responding on my phone lol

Reply:
@Omikorin it's also a default setting for json and `indentWidth` in the config. No need to add.

And the indentation of this turbo config was not changed locally. It is definitely showing a width of `2` despite it not looking that way here.

Might it be because it's in `tab` style and not `space` style?

Reply:
I fixed it for you

Reply:
Oh, thanks! Didn't think about that option